consensus/istanbul/validator/weighted.go

```diff
@@ -844,7 +845,7 @@ func calcWeight(weightedValidators []*weightedValidator, stakingAmounts []float6
 	localLogger.Debug("calculation weight finished")
 }
 
-func (valSet *weightedCouncil) refreshProposers(seed int64, blockNum uint64) {
+func (valSet *weightedCouncil) refreshProposers(seed int64, blockNum uint64, rules params.Rules) {
```

Comment:
I don't think this parameter is used anymore. Please remove it.

Reply:
Thank you for your review.
Reflected in `b970e0dd7bc1a7181f649d6462ce3aee6a14153f`.